test/accessibility/apps/discover.ts

```diff
@@ -21,21 +21,14 @@ import { FtrProviderContext } from '../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'discover', 'header', 'share', 'timePicker']);
-  const retry = getService('retry');
   const a11y = getService('a11y');
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
   const inspector = getService('inspector');
-  const docTable = getService('docTable');
-  const filterBar = getService('filterBar');
-  const TEST_COLUMN_NAMES = ['@message'];
-  const TEST_FILTER_COLUMN_NAMES = [
-    ['extension', 'jpg'],
-    ['geo.src', 'IN'],
-  ];
-
-  // Failing: See https://github.com/elastic/kibana/issues/59975
-  describe.skip('Discover', () => {
+  const testSubjects = getService('testSubjects');
+  const TEST_FILTER_COLUMN_NAMES = [['extension'], ['geo.src']];
```

Comment:
Sine this no longer needs an array of arrays it problem makes sense to trim this back down to (I also took the `FILTER` out of the name since it's not being used for that purpose in this test);
```
const TEST_COLUMN_NAMES = ['extension', 'geo.src'];
```
and then in the test;
```
for (const columnName of TEST_COLUMN_NAMES) {
```